tsdb/chunkenc/histogram.go

```diff
@@ -86,28 +87,14 @@ const (
 	// UnknownCounterReset means we cannot say if this chunk was created due to a counter reset or not.
 	// An explicit counter reset detection needs to happen during query time.
 	UnknownCounterReset CounterResetHeader = 0b00000000
+	// CounterResetHeaderMask is a mask to get the counter reset bits.
+	CounterResetHeaderMask byte = 0b11000000
```

Comment:
Go style: I think this should be on its own `const` line. The existing `const` block are all the possible values for the (pseudo) enum `CounterResetHeader`.